app/views/users/sessions/verify_2fa.html.erb

```diff
@@ -1,4 +1,15 @@
 <h1>Two-factor authentication</h1>
+
+<script>
+  addEventListener('load', () => {
```

Comment:
Minor nit: `window.addEventListener` :) It's not wrong, but best use global explicitly